From 6356fa37d355c8f9b634ea6f568a9ca34afad819 Mon Sep 17 00:00:00 2001
From: Kostya Esmukov <kostya@esmukov.ru>
Date: Sun, 24 Nov 2019 23:44:07 +0300
Subject: [PATCH 3/9] DockerOperator: disable `template_ext`

Otherwise

```
command:
- some_bin.sh
```

... is interpreted as "load jinja2 template `some_bin.sh`"
instead of "call `some_bin.sh` command".
---
 airflow/operators/docker_operator.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/airflow/operators/docker_operator.py b/airflow/operators/docker_operator.py
index cb335d3dd..47ec3762f 100644
--- a/airflow/operators/docker_operator.py
+++ b/airflow/operators/docker_operator.py
@@ -125,7 +125,7 @@ class DockerOperator(BaseOperator):
     :type tty: bool
     """
     template_fields = ('command', 'environment', 'container_name')
-    template_ext = ('.sh', '.bash',)
+    template_ext = tuple()
 
     @apply_defaults
     def __init__(
-- 
2.25.0

